### DISCUSSION TITLE
SQL Server license

### DISCUSSION THREAD

**Discussion by GioviQ** [microsoft/aspire#4343] [General]
Downloading SQL Server in a container gives you the (wrong) idea that it is free to use. What about the license instead?

**Comment by Lindsay-Mathieson** [comment]
Its SQL Express which is free to use for personal development

**Reply by GioviQ** [reply]
Can you post the link where you read that the container has SQL Server Express inside? Thanks

**Reply by GioviQ** [reply]
I found https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables
MSSQL_PID setting
